converters/fromZigbee.js

```diff
@@ -430,6 +430,53 @@ const converters = {
             }
         },
     },
+    level_config: {
+        cluster: 'genLevelCtrl',
+        type: ['attributeReport', 'readResponse'],
+        convert: (model, msg, publish, options, meta) => {
+            const result = {'level_config': {}};
+
+            // onOffTransitionTime - range 0x0000 to 0xffff - optional
+            if (msg.data.hasOwnProperty('onOffTransitionTime') && (msg.data['onOffTransitionTime'] !== undefined)) {
+                result.level_config.onOffTransitionTime = msg.data['onOffTransitionTime'];
```

Comment:
Maybe `level` should be renamed to `brightness` here, I guess users aren't familiar with the `level` term.

Reply:
I've now removed it from the expose for bulbs and manually added it for the Ubisys D1.

I kind of want to leave it at level_config, as the tech manual the user was using also refers to it as the level cluster... and tbh, if you want to use this, you probably know what you're doing and we should dumb it down?

Reply:
Agree